src/Collections/Sites.php

```diff
@@ -190,41 +179,74 @@ public function findUuidByName($name)
     }
 
     /**
+     * Get the site's UUID given either it's site name or it's UUID
+     *
+     * @param $id
+     * @return string
+     */
+    public function findUuid($id)
+    {
+        // Make the search case insensitive.
+        $id = strtolower($id);
+
+        // If it LOOKS like a uuid, then we assume it is. If a user names a site with a UUID he forfeits his right
```

Comment:
Yeah. Not great here are the options for this:

1. Use the inclusive but grammatically non-traditional 'they/them'
2. Use the somewhat inclusive but grammatically awkward 'he/she', 'him/her'.
3. Restructure the sentence to be legitimately plural.
4. Switch to the first person.

But of course the correct answer is number 4:

Remove the sentence altogether because it's a bit snarky and not particularly useful.